xontrib/voxapi.py

```diff
@@ -22,6 +23,10 @@
 # FIXME: Is there a better way?
 from xonsh.events import events
 
+VOX_DEFAULT_INTERPRETER = builtins.__xonsh__.env.get(
+    "VOX_DEFAULT_INTERPRETER", sys.executable
+)
```

Comment:
It is usually better to pull the env vars out right before they are used, rather than at module-level scope.